README.rdoc

```diff
@@ -39,7 +43,7 @@ will maintain stability in master.
 == RESOURCES
 
 * {Website}[http://rubygems.org/]
-* {Documentation}[http://rubygems.rubyforge.org/rubygems-mirror/README_rdoc.html]
+* {Documentation}[http://rubygems.rubyforge.org/rubygems-mirror/README.rdoc]
```

Comment:
This url doesn't actually exist either, please update to `https://github.com/rubygems/rubygems-mirror/README.rdoc`